src/cli/cli_coap_secure.cpp

```diff
@@ -147,6 +173,45 @@ template <> otError CoapSecure::Process<Cmd("set")>(Arg aArgs[])
     return error;
 }
 
+/**
+ * @cli coaps start
+ * @code
+ * coaps start
+ * Done
+ * @endcode
+ * @code
+ * coaps start false
+ * Done
+ * @endcode
+ * @code
+ * coaps start 8
+ * Done
+ * @endcode
+ * @cparam coaps start [@ca{check-peer-cert} | @ca{max-conn-attempts}]
+ * The `check-peer-cert` parameter determines if the peer-certificate check is
+ * enabled (default) or disabled.
+ * The `max-conn-attempts` parameter sets the maxinum number of allowed
```

Comment:
```suggestion
 * The `max-conn-attempts` parameter sets the maximum number of allowed
```

Reply:
@jwhui thanks good catch, fixed the typo.